packages/chakra-ui-core/src/CToast/CToast.js

```diff
@@ -47,47 +47,56 @@ const CToast = {
     },
```

Comment:
Hey I think you have a global prettier setting modifying the files in your editor. You might want to check the `Formatting` tab on the taskbar. Remember to tun this off. Think you can resolve this?